ethereum/rpc/backend/backend.go

```diff
@@ -148,9 +148,13 @@ func (e *EVMBackend) EthBlockFromTendermint(
 
 		gasUsed += gas
 
-		msg, isEthTx := tx.(*evmtypes.MsgEthereumTx)
+		if len(tx.GetMsgs()) != 1 {
+			// TODO: eventually support Cosmos txs in the block
```

Comment:
more description?

Reply:
actually I removed this comment because it is not really clear @fedekunze ?